--- cpplint.py.default	2014-06-11 22:54:37 +0900
+++ cpplintDir.py	2014-06-12 10:35:48 +0900
@@ -1,4 +1,5 @@
-#!/usr/bin/python
+#!python
+#/usr/bin/python
#
# Copyright (c) 2009 Google Inc. All rights reserved.
#
@@ -4649,7 +4650,7 @@
  if message:
    sys.exit('\nFATAL ERROR: ' + message)
  else:
-    sys.exit(1)
+    sys.exit(0)


def PrintCategories():
@@ -4732,7 +4733,7 @@


def main():
-  filenames = ParseArguments(sys.argv[1:])
+  baseDirName = ParseArguments(sys.argv[1:])

  # Change stderr to write with replacement characters so we don't die
  # if we try to print something containing non-ASCII characters.
@@ -4742,11 +4743,13 @@
                                         'replace')

  _cpplint_state.ResetErrorCounts()
-  for filename in filenames:
-    ProcessFile(filename, _cpplint_state.verbose_level)
+  for dname in baseDirName:
+    for root, dirs, files in os.walk(dname):
+      for fname in files:
+        ProcessFile(os.path.join(root, fname),
_cpplint_state.verbose_level)
  _cpplint_state.PrintErrorCounts()

-  sys.exit(_cpplint_state.error_count > 0)
+  sys.exit(0)


if __name__ == '__main__':

